res/cmake/def.cmake

```diff
@@ -1,116 +1,42 @@
 
 
-if(testMPI)
-  if (NOT DEFINED PHARE_MPI_PROCS)
-    set(PHARE_MPI_PROCS 2)
-  endif()
-else()
-  if (NOT DEFINED PHARE_MPI_PROCS)
-    set(PHARE_MPI_PROCS 1)
-  endif()
-endif()
+set (PHARE_FLAGS ${PHARE_FLAGS})
+set (PHARE_WERROR_FLAGS ${PHARE_FLAGS} ${PHARE_WERROR_FLAGS})
+set (PHARE_PYTHONPATH "${CMAKE_BINARY_DIR}:${CMAKE_SOURCE_DIR}/pyphare")
 
 set (PHARE_HAS_HIGHFIVE "0")
 if (HighFive)
  set (PHARE_HAS_HIGHFIVE "1")
 endif()
 
-# Pybind errors with clang, it is default in GCC
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-  set (CMAKE_CXX_FLAGS -fsized-deallocation)
+if (NOT DEFINED PHARE_MPI_PROCS)
```

Comment:
shouldn' that rather be in the test config file?

Reply:
it's something used by testing, but it's it only used by testing?
currently maybe, so maybe yes it should be